lib/BaseModel.js

```diff
@@ -4,8 +4,9 @@ var spatialReference = require('./SpatialReference')
 /**
  * Uses the module pattern for clean inheritance in models
  * exposes shared functionality across providers, typically things that require central code in koop
+ * TODO: refactor so that koop app isn't passed in at all
  *
- * @param {Object} koop
+ * @param {Object} koop - instance of koop middleware app
  */
 function Model (koop) {
```

Comment:
Made a note above -- in a perfect future we're not giving the model constructor a copy of the entire koop app.